changelog/unreleased/10406

```diff
@@ -0,0 +1,6 @@
+Enhancement: Disable dip scale rounding on Windows
```

Comment:
```suggestion
Enhancement: Disable DPI scaling factor rounding on Windows
```

It's definitely DPI. I also feel "scaling factor" is a more accurate, more user friendly term. But I saw Qt is using "scale factor", so that's probably fine.